code/game/machinery/doors/firedoor.dm

```diff
@@ -231,7 +253,19 @@
 	var/datum/merger/merge_group = GetMergeGroup(merger_id, merger_typecache)
 	for(var/obj/machinery/door/firedoor/buddylock as anything in merge_group.members)
 		buddylock.activate(code)
-
+/**
+ * Begins deactivation process of us and our neighbors.
+ *
+ * This proc will call reset() on every fire lock (including us) listed
+ * in the merge group datum. sets our alarm type to null, signifying no alarm.
+ */
+/obj/machinery/door/firedoor/proc/start_deactivation_process()
+	alarm_type = null
+	soundloop.stop()
+	is_playing_alarm = FALSE
+	var/datum/merger/merge_group = GetMergeGroup(merger_id, merger_typecache)
+	for(var/obj/machinery/door/firedoor/buddylock as anything in merge_group.members)
+		buddylock.reset()
```

Comment:
What happens if one of our friends still has a valid reason to be closed?

Reply:
they all open, but that one will close again on its own

Reply:
That's cringe, can we do anything about it?

Reply:
> That's cringe, can we do anything about it?

perhaps? I mean it basically opens with the rest then instantly closes again

Reply:
lemon I dont know if its really a big deal functionally, they open for a split second then instantly close again

Reply:
I really dislike how it looks, I find it kinda silly. There are ways to get around this, if you're up to it I can walk you through them

Reply:
Im not closing this because of the epic quote but Im not sure if this is still an active issue

Reply:
This is ok now